### PR TITLE
Design

### PR DESCRIPTION
**Pull request by larisabainton**

### CODE REVIEW

File: app/assets/stylesheets/colors.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+$pink: #f6c3ef;
+$dark-grey: #363535;
+$light-grey: #D3D3D3;
```
Comment (nwalberts) [inline app/assets/stylesheets/colors.scss]: address this

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+  :visited {
+    text-decoration: none;
+    color: $black;
+  }
+}
+
+.flash {
+  background-color: $dark-grey;
+  color: $white;
+  padding: 1rem;
```
Comment (nwalberts) [inline app/assets/stylesheets/application.scss]: not required, but suggested you reorder

File: app/assets/stylesheets/show.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+  p {
+    margin: 1rem;
+  }
+
+  .edit-delete-recipe {
+    float: right;
+    margin-top: -5.5rem;
+    font-family: Quattrocento;
+    font-color: $medium-grey
```
Comment (nwalberts) [inline app/assets/stylesheets/show.scss]: address this

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -1,2 +1,29 @@
...
+@import './show';
+@import './index';
+@import './new';
 
+body {
+  background-color: $pink;
+  font-family: Arial;
+
+  :link {
+    text-decoration: none;
```
Comment (jraenorris) [inline app/assets/stylesheets/application.scss:13]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -1,2 +1,29 @@
...
+    color: $black;
+  }
+
+  :visited {
+    text-decoration: none;
```
Comment (jraenorris) [inline app/assets/stylesheets/application.scss:30]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/assets/stylesheets/index.scss [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  :visited {
+    color: $dark-grey;
+  }
+}
+
+.recipe-index {
+  background-color: $white;
+  padding: 1rem;
```
Comment (jraenorris) [inline app/assets/stylesheets/index.scss]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/assets/stylesheets/index.scss [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  border: $medium-grey solid 2px;
+
+  h2 {
+    font-size: 2rem;
+  }
+
+  .recipe-teaser {
+    padding: 1rem;
```
Comment (jraenorris) [inline app/assets/stylesheets/index.scss]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/assets/stylesheets/index.scss [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    border-bottom: $light-grey solid;
+  }
+}
+
+.add-recipe {
+  color: $white;
```
Comment (jraenorris) [inline app/assets/stylesheets/index.scss:60]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/assets/stylesheets/show.scss [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+    font-size: 1.3rem;
+    margin: 1rem;
+  }
+
+  p {
+    margin: 1rem;
+  }
+
+  .edit-delete-recipe {
+    float: right;
```
Comment (jraenorris) [inline app/assets/stylesheets/show.scss:33]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/models/review.rb [hunk 10]
```diff
@@ -3,5 +3,8 @@ class Review < ActiveRecord::Base
   belongs_to :user
 
   validates :rating, numericality: {only_integer: true},
-  inclusion: {in: 1..5, message: "Rating must be between 1 - 5"}
+  inclusion: {
```
[the remark this answers is not in the document]
Comment (jraenorris) [reply app/models/review.rb]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

[the remark this answers is not in the document]
Comment (jraenorris) [reply app/models/review.rb]: Address all hound violations. If you update the .yml to address them, make a reply on the hound comments

File: app/views/recipes/show.html.erb [hunk 11]
```diff
@@ -1,44 +1,78 @@
-<%= @recipe.title %>
-<% full_name = @recipe.user.first_name + ' ' + @recipe.user.last_name %>
-Created by: <%= link_to full_name, user_path(@recipe.user.id) %>
+<<<<<<< HEAD
```
Comment (jraenorris) [inline app/views/recipes/show.html.erb]: Looks like this is the remains of a merge conflict?

File: app/views/recipes/show.html.erb [hunk 12]
```diff
@@ -1,44 +1,78 @@
...
-  <%= link_to "Edit Recipe", edit_recipe_path(@recipe) %>
-  <%= link_to "Delete Recipe", @recipe, method: :delete, data: { confirm: "You sure?" }  %>
-<% end %>
+
+  <div class="add-review">
+    <h3>Add a Review!</h3>
+    <%= form_for [@recipe, @review] do |f| %>
+    <%= f.label :rating %>
+    <%= f.text_field :rating %>
+    <br>
```
Comment (jraenorris) [inline app/views/recipes/show.html.erb:69]: It's better practice to nest things in `<p>` tags rather than tossing in `<br />` tags here and there

**Review by jraenorris — CHANGES_REQUESTED** [review]
Did you mean to upload the cookie image?

File: app/views/recipes/show.html.erb (hunk 12 above)
Comment (enthusiastick) [reply app/views/recipes/show.html.erb:69]: Also, if you're going to use `<br />` tags (which, Jesse is right, you should avoid), at least make them self-closing.

**Comment by enthusiastick** [comment]
Train that hound.

selfie-0
[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

**Pull request merged**